tests/integration/record_benchmark.py

```diff
@@ -166,11 +166,15 @@ def data_container(data: dict):
             else:  # Nightly build
                 data["djlNightly"] = "true"
                 data["image"] = container[:-len("-nightly")]
-        if "text-generation-inference" in container:
+        elif "text-generation-inference" in container:
             data["modelServer"] = "TGI"
             version = container.split(":")[1]
             if not version.startswith("sha"):
                 data["tgiVersion"] = version
+        elif "suzuka" in container:
```

Comment:
Rather than having `sazuka` and `other`, why not use `container.split("/", 1)[1]`? It will be a solid value that will work for sazuka and future use cases as well